package.json

```
@@ -16,7 +16,9 @@
     "next-connect": "0.9.1",
     "react": "16.12.0",
     "react-dom": "16.12.0",
+    "react-icons": "^4.2.0",
     "redoc": "2.0.0-rc.50",
+    "sass": "^1.32.12",
```

Pelo que entendi eles estão "pinando" as versões, então teria de modificar para 
```suggestion
    "sass": "1.32.12",
```